src/app/calculator/fans/fan-analysis/fan-analysis-form/plane-data-form/pressure-readings-form/pressure-readings-form.component.ts

```diff
@@ -93,8 +88,31 @@ export class PressureReadingsFormComponent implements OnInit {
     this.save();
   }
 
+  setStaticPressure() {
+    let row;
```

Comment:
add types and a comment or two about what's being done here. (I should have added earlier -sorry)